internal/ent/schema/note.go

```diff
@@ -96,6 +96,11 @@ func (n Note) Edges() []ent.Edge {
 			edgeSchema: InternalPolicy{},
 			ref:        "comments",
 		}),
+		uniqueEdgeFrom(&edgeDefinition{
```

Comment:
assuming you'll address the trust center access in a follow up? i think that just involves setting the trust center as a parent and setting up some fga tuples